src/lib/import-export/export/types.ts

```diff
@@ -29,3 +29,17 @@ export interface BackupCreateProgressEventData {
 }
 
 export type NewExporter = new ( options: ExportOptions ) => Exporter;
+
+export interface StudioJson {
+	phpVersion: string;
+	wordpressVersion?: string;
+	siteUrl: string;
+	plugins: StudioJsonPluginOrTheme[];
+	themes: StudioJsonPluginOrTheme[];
+}
+
+export interface StudioJsonPluginOrTheme {
+	name: string;
+	status: 'active' | 'inactive';
+	version: string;
+}
```

Comment:
@myhro - we can reuse that typing for https://github.com/Automattic/jetpack-backup-team/issues/694 / https://github.com/Automattic/jetpack-backup-team/issues/693

Reply:
Makes sense to me. Thanks for pointing it out!